src/Microsoft.AspNet.Identity/RoleManager.cs

```diff
@@ -20,13 +21,17 @@ namespace Microsoft.AspNet.Identity
     public class RoleManager<TRole> : IDisposable where TRole : class
     {
         private bool _disposed;
-        private HttpContext _context;
+        private readonly HttpContext _context;
 
         /// <summary>
-        /// Constructor
+        ///     Constructor
         /// </summary>
         /// <param name="store">The IRoleStore commits changes via the UpdateAsync/CreateAsync methods</param>
-        /// <param name="roleValidator"></param>
+        /// <param name="roleValidators">IEnumerable of role validators</param>
+        /// <param name="keyNormalizer">user property normalizers</param>
+        /// <param name="errors">IdentityErrorDescribers</param>
```

Comment:
So unit testing is a valid scenario, MVC and Http both invested effort to make their classes new()able for ease of use, so I actually don't think this has been completed settled yet. I wouldn't rathole on this now.  Somewhere on my todo list is to revisit unit testing/etc